kodilla-collections/src/main/java/com/kodilla/collections/sets/homework/Stamp.java

```diff
@@ -0,0 +1,49 @@
+package com.kodilla.collections.sets.homework;
+
+import java.util.Objects;
+
+public class Stamp {
+    private String stampName;
+    private String stampDimensions;
+    private String stamped;
+
+    public Stamp(String stampName, String stampDimensions, String  stamped){
+        this.stampName = stampName;
+        this.stampDimensions = stampDimensions;
+        this.stamped=stamped;
+    }
```

Comment:
Uwaga do sekcji z zadania
```
Klasa powinna mieć pola na nazwę oraz wymiary znaczka, a także informację o tym, czy znaczek jest ostemplowany czy nie
```
Nazwa - String jest ok
Wymiary znaczka - to powinien byc int/double albo para intow/double'i
stamped - tak, albo nie osteplowany - czyli boolean bardziej pasuje